tests/gnrc_tcp/tests/test_03-send_data.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2018   Simon Brummer <simon.brummer@posteo.de>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+import socket
+import threading
+from shared_func import getHostLinkLocalAddr
+from shared_func import getRiotInterfaceId
+from shared_func import setupTcpBuffer
+from shared_func import addDataToTcpBuffer
+
+HOST_IF = 'tap0'
+SERVER_PORT = 15000
+
+def tcpServer(expectedData):
+    sock = socket.socket(socket.AF_INET6, socket.SOCK_STREAM)
+    sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+    sock.bind(('::', SERVER_PORT))
+    sock.listen(1)
```

Comment:
This get's repeated pretty often. You could just have a helper module for that, added to `tests/gnrc_tcp/tests/` in a non-executable python file.